Model/ConfigProvider/Account.php

```diff
@@ -235,4 +236,389 @@ protected function getMethodConfigProvider($paymentMethod)
 
         return $this->methodConfigProviderFactory->get($methodCode);
     }
+
+
+    /**
+     * {@inheritdoc}
```

Comment:
I see a lot of @inheritdoc but is there actually documentation available?
I'd love to see maybe also some functional documentation such as what settings this actually relates to. Or what it does.